### PR TITLE
bsunanda:Phase2-ft11 Update README file in Configuration/Geometry and add required files for FastTime Reco Geometry

### PR DESCRIPTION
**Pull request by bsunanda**
Updating the README file; include fast time RECO geometry in I2; making room for Geometry tester for FastTimer. This is an addition after PR# 16562

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
Creation of a geometry with I2 can be done with this only after PR 16562 was integrated

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
hi @bsunanda -please update the title to be something more descriptive for the release notes. Thanks!

**Comment by bsunanda** [comment]
@davidlange6 Done that

**Comment by davidlange6** [comment]
+1

File: Configuration/Geometry/scripts/dict2023Geometry.py [hunk 1]
```diff
@@ -496,6 +496,10 @@
             'from Geometry.HGCalCommonData.fastTimeParametersInitialization_cfi import *',
             'from Geometry.HGCalCommonData.fastTimeNumberingInitialization_cfi import *',
         ],
+        "reco" :[
+            'from Geometry.CaloEventSetup.FastTimeTopology_cfi import *',
+            'from Geometry.HGCalGeometry.FastTimeGeometryESProducer_cfi import *',
+        ],
```
Comment (kpedro88) [inline Configuration/Geometry/scripts/dict2023Geometry.py]: @bsunanda you did not actually regenerate the geometry cffs after this change. Was that intentional?

**Pull request merged**